extensions/wikia/AdEngine/js/spec/slot/resolvedState.spec.js

```diff
@@ -2,128 +2,140 @@
 describe('ext.wikia.adEngine.slot.resolvedState', function () {
 	'use strict';
 
-	var BIG_IMAGE = 'bigImage.png',
+	function createCorrectParams() {
+		return {
+			imageSrc: BIG_IMAGE,
+			aspectRatio: ASPECT_RATIO,
+			resolvedState: {
+				aspectRatio: RESOLVED_STATE_ASPECT_RATIO,
+				imageSrc: RESOLVED_IMAGE
+			},
+			backgroundImage: {
+				src: DEFAULT_IMAGE
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "DEFAULT_IMAGE" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)